pkg/server/app/models.go

```diff
@@ -237,3 +249,33 @@ func unsetEnvVars(app *App, evs []string) {
 		}
 	}
 }
+
+func newListResponse(list []*List) *appb.ListResponse {
+	if list == nil {
+		return nil
+	}
+
+	appNames := []*appb.ListResponse_App{}
+
+	for _, item := range list {
+		if item == nil {
+			continue
+		}
+		var tmp []string
+		for _, elt := range item.Addresses {
+			tmp = append(tmp, elt.Hostname)
```

Comment:
Hummm. I did not know this difference between them. It is nice to know that! I will include an update in my code. Tks